front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -107,6 +108,12 @@ const StdcmView = () => {
     }
   }, [isDebugMode]);
 
+  useEffect(() => {
+    if (isCanceled) {
+      setShowBtnToLaunchSimulation(true);
+    }
+  }, [isCanceled]);
```

Comment:
Instead of this `useEffect()`, we could pass `showBtnToLaunchSimulation={showBtnToLaunchSimulation || isCanceled}` in the JSX below. This would make it easier to reason about the data flow.

Reply:
oh good idea